com.xargon.retrodeck.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=RetroDECK
+GenericName=Emulation bundle for Steam Deck
+Type=Application
+Comment=All you need for emulation on Steam Deck
+Icon=com.xargon.retrodeck
+Exec=
```

Comment:
there should be an executable here.

Reply:
That is generated by the flatpak-builder.

I am referring to this: https://docs.flatpak.org/en/latest/freedesktop-quick-reference.html?highlight=exec#desktop-files

```
If you want the --command argument to be omitted from the flatpak run command in the generated desktop file, you can leave the Exec value in the source desktop file empty
```